packages/commonwealth/server/migrations/20221115210648-migrate-snapshot-data-to-new-table.js

```diff
@@ -0,0 +1,57 @@
+'use strict';
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    /**
+     * Add altering commands here.
+     *
+     * Example:
+     * await queryInterface.createTable('users', { id: Sequelize.INTEGER });
+     */
+    return queryInterface.sequelize.transaction(async (t) => {
```

Comment:
Why so many migrations, that seem to be operating on tables created immediately prior? We should group these up a little more clearly so the data path is more obvious.